docs/TLSsupport.md

```diff
@@ -0,0 +1,31 @@
+# Support Transport Layer Security (TLS) for Communication with Connectors
+
+## Requirements
+- TLS communication is an optional feature, if it is not turned ON, the regular TCP/HTTP communication should be available
+- TLS can be extended to mutual TLS
+- The system should allow setting required TLS parameters, Certificates, Certificate Authorities (CA).
+- If CA are not set, the default CA will be used. Due to close communication between Fybrik manager and connectors, 
+  explicit setting CA will **replace** the default goLang or Java CA settings. Optionally, they can be merged.
+- Optionally the system will allow setting the minimal supported TLS protocol, possible values are:
+  TLS 1.0 ([RFC 2246](https://datatracker.ietf.org/doc/html/rfc2246)), 
+  TLS 1.1 ([RFC 4346](https://datatracker.ietf.org/doc/html/rfc4346)), 
+  TLS 1.2 ([RFC 5246](https://datatracker.ietf.org/doc/html/rfc5246)) and 
+  TLS 1.3 (RFC [8446](https://datatracker.ietf.org/doc/html/rfc8446))
+- The current implementation will not allow changing the default cipher suites. (golang allows defining a list of enabled 
+TLS 1.0–1.2 cipher suites. TLS 1.3 cipher suites are not configurable.)
+- In order to allow automatic certificates renew or revoke, [cert-manager](https://cert-manager.io/) will be used to 
+  manage the certificates.
```

Comment:
I do not think cert-manager is part of the requirement. the requirement is to store the certificates in secrets.

Reply:
I rephrased the sentence, please take a look